src/scheduler.h

```diff
@@ -72,6 +73,14 @@ class Scheduler {
   // Return the number of in-flight inferences tracked by the scheduler.
   virtual size_t InflightInferenceCount() = 0;
 
+  // Update the scheduler during a model update (i.e. instances). This function
```

Comment:
> This function should block any new inference requests into 'Enqueue()' when ready (i.e. sequence models)

The comment seems ambiguous. Can you make it more accurate?

Reply:
Yes. Updated: [Remove lock returned from scheduler update](https://github.com/triton-inference-server/core/pull/207/commits/092a2581d221f8c78d1019a092682a1ada29adce)